daemon/daemon_main.go

```diff
@@ -810,14 +820,51 @@ func initEnv(cmd *cobra.Command) {
 	}
 }
 
+// setHostDeviceWhenReady sets the given ifaceName as the HostDevice. If
+// ifaceName is not found, then it will wait forever until the device is
+// created before it assign it to the hostDevice.
+func setHostDeviceWhenReady(ifaceName string) {
```

Comment:
What happens when the interface goes down, or when it gets re-created by a CNI plugin? Shouldn't we monitor the interface and re-install the BPF programs?